package-lock.json

```diff
@@ -2535,7 +2535,8 @@
         "ansi-regex": {
```

Comment:
you don't need to change this but in the future you shouldn't need to change package.lock.json if you didn't change the dependencies. To restore without changing the dependencies you can run `npm ci` :smile:

Reply:
Thanks for the info on this!